libraries/compute-utils/function-views/src/function-view.ts

```diff
@@ -192,13 +192,20 @@ export abstract class FunctionView extends DG.ViewBase {
    * @stability Stable
   */
   protected async loadFuncCallById() {
+    if (this.initValue instanceof DG.FuncCall) {
+      // next tick is needed to run funcCallReplaced before building UI
+      await new Promise(resolve => setTimeout(resolve, 0));
```

Comment:
I can't get point of this line. 
`this.funcCallReplaced` only has meaning when UI is already built. Using this event, we can update UI when `this.funcCall` is replaced. By the moment of this line, UI is not built yet. 

Tried to comment this line and run my code - it works properly.

Reply:
Ah, got it